src/Microsoft.AspNetCore.Hosting/Startup/ExceptionPage/Views/ErrorPage.Generated.cs

```diff
@@ -609,7 +609,37 @@ public ErrorPage()
 #line default
```

Comment:
Ok, figured out. The files need to be named either .designer.cs or .feature.cs (Per https://github.com/github/linguist/blob/c471990aa3ced30bb6edb25c3a3530ab90066a36/lib/linguist/generated.rb#L203 \ https://github.com/github/linguist/blob/c471990aa3ced30bb6edb25c3a3530ab90066a36/lib/linguist/generated.rb#L214) which would cause GitHub to suppress diffs for it by default.

.Designer.cs might be better name since .Feature doesn't necessarily make a lot of sense.